File: pset6/mario.py
import cs50

def main():
    print("Height: ", end=' ')
    x = cs50.get_int()
    
    while x<0 or x>23:
        print("Height: ", end=' ')
        x = cs50.get_int()
        if x>0 and x<23:
            break
    
    for i in range(x):
        yell((x - i) - 1) 
        cough(i + 1)
        print("  ", end="")
        cough(i + 1)
        jell((x - i) - 1)
        print("")
        
   # print ("")
   
def yell(n):
        for i in range (n):
            print(" ",end="")
    
def cough(n):
        for i in range (n):
            print("#",end="")

def jell(n):
       for i in range (n):
          print(" ",end="")
        
    
if __name__ == "__main__":
    main()
    

File: pset1/mario.c
#include <stdio.h>
#include <cs50.h>

void cough(int n);
void yell (int n);
int rows;
int main(void)

{
    

{
    printf("Height:");
    rows = get_int();
}

while (rows<0 || rows > 23)
{
    printf("Height:");
    rows = get_int();
}

for (int i = 0; i < rows; i++)

{
    yell((rows -i)-1);
    cough(i + 1);
    printf("  ");
    cough(i+1);
    yell((rows -i)-1);
    printf("\n");
    
    
    
}


}


void cough(int n)
{
    for (int i = 0; i < n; i++)
    {
        printf("#");
        
    }


    
}
void yell(int n)
{
    for (int i = 0; i < n; i++)
    {
        printf(" ");
        
    }
}

